language/move-vm/natives/src/vdf.rs

```diff
@@ -11,17 +11,22 @@ use move_vm_types::{
     natives::function::{native_gas, NativeContext, NativeResult},
     values::{Reference, Value},
 };
-use std::collections::VecDeque;
+use std::{collections::VecDeque, time::Instant};
 use std::convert::TryFrom;
 use move_binary_format::errors::{PartialVMError, PartialVMResult};
 use smallvec::smallvec;
+use crate::counters::MOVE_VM_NATIVE_VERIFY_VDF_LATENCY;
 
 /// Rust implementation of Move's `native public fun verify(challenge: vector<u8>, difficulty: u64, alleged_solution: vector<u8>): bool`
 pub fn verify(
     context: &impl NativeContext,
     _ty_args: Vec<Type>,
     mut arguments: VecDeque<Value>,
 ) -> PartialVMResult<NativeResult> {
+    // temporary logging.
+    let start_time = Instant::now();
+    let metric_timer = MOVE_VM_NATIVE_VERIFY_VDF_LATENCY.start_timer();
+
```

Comment:
@intuitiveminds could we also get a counter here as well? I'm interested in how many requests to this function exist in a period, as well as the durantion. Thakns!

Reply:
I added two counters here, one for the overall number of submitted requests in a period and one for failed requests because of illegal arguments.

Reply:
thanks